src/main/java/gregtech/api/metatileentity/multiblock/MultiblockControllerBase.java

```diff
@@ -115,6 +116,10 @@ public final int getLightValue() {
         return blockWorldState -> ArrayUtils.contains(block, blockWorldState.getBlockState().getBlock());
     }
 
+    public static Predicate<BlockWorldState> isAirPredicate() {
+        return blockWorldState -> blockWorldState.getBlockState().getBlock().isAir(blockWorldState.getBlockState(),null,null);
```

Comment:
passing nulls to method excepting world-sensitive info is not how it works.

Reply:
These parameters are advertised as nullable and not used to check a block is air
![image](https://user-images.githubusercontent.com/1111474/49098356-faa14f00-f26e-11e8-9be2-37fbabacb03f.png)